docs/stack.md

```diff
@@ -1,42 +1,64 @@
-## Stack
+# Stack
 
-### HTML
+## HTML
 
 HTML for the majority of the site is generated from Django/Wagtail templates and components.
 
-### Frontend styleguide
+## Frontend styleguide
 
 This site has a [styleguide](https://foundation.mozilla.org/en/style-guide/), which can be refrenced for frontend development to note intended usage of exisitng styles and components.
 
-### CSS
+## CSS
 
-CSS is generated from [Sass](http://sass-lang.com/). Styles are based on the mofo-bootstrap theme found under `source/sass/mofo-bootstrap`.
+CSS is currently very messy.
+We are using a mix of custom [Sass](http://sass-lang.com/), mixed with the mofo-bootstrap theme found under `source/sass/mofo-bootstrap` while we also trying to move away from Bootstrap to [Tailwind CSS](https://tailwindcss.com/).
```

Comment:
referencing the `tailwind.config.js` is probably the most important reference of how we are using tailwind on the site.

Reply:
Do you have a suggestion what I should mention about the config? I hardly ever touch it, so I am not sure what to say about it.

Reply:
So I think it is important to look at in case you are using tokens that have been manipulated from the original tailwind ones. At the moment we have changed colours,spacing,breakpoints and fonts. Since we link the site, I want a way to mention that they should also keep the config in mind so someone is confused that yellow-60 isn't working as intended or if they create custom classes that have already been made.

I think just saying heads up we have a config that has changed x,y,z and includes our plugin usage would be great!

Reply:
Added a comment in 40a87c5a22c7d8624f422e88794174d19166286a